modules/_labs/panel/react/README.md

```diff
@@ -0,0 +1,42 @@
+# Canvas Kit React Panel
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/_labs/README.md">
+  <img src="https://img.shields.io/badge/LABS-alpha-orange" alt="LABS: Alpha" />
+</a>  This component is work in progress and currently in pre-release.
+
+
+
+## Installation
```

Comment:
You can only install by adding `@workday/canvas-kit-labs-react-panel`.

I'll add an issue to track the fact that the create-component script isn't doing this for us.

Reply:
Created #279 to track.